src/components/sbb-select/sbb-select.scss

```diff
@@ -0,0 +1,156 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+// Fixes the gap between the origin and the overlay by creating conjuction
+// corners based on the origin element border radius
+@include sbb.overlay-gap-fix(var(--sbb-options-panel-border-radius));
+
+:host {
+  @include sbb.optionsPanelOverlayHost;
+
+  --sbb-options-panel-internal-z-index: var(--sbb-select-z-index, var(--sbb-overlay-z-index));
+}
+
+:host([data-state='closed']) {
+  --sbb-options-panel-visibility: hidden;
+}
+
+:host([data-state='opening']) {
+  --sbb-options-panel-animation-name: open;
+}
+
+:host([data-state='closing']) {
+  --sbb-options-panel-animation-name: close;
+}
+
+:host([data-state='opened']),
+:host([data-state='opening']) {
+  --sbb-options-panel-gap-fix-opacity: 1;
+}
+
+:host([data-options-panel-position='below']) {
+  --sbb-options-panel-animation-transform: translateY(
+    calc((var(--sbb-options-panel-origin-height) / 2) * -1)
+  );
+}
+
+:host([data-options-panel-position='above']) {
+  --sbb-options-panel-options-border-radius: var(--sbb-options-panel-border-radius)
+    var(--sbb-options-panel-border-radius) 0 0;
+  --sbb-options-panel-gap-fix-top: var(--sbb-options-panel-max-height);
+  --sbb-options-panel-gap-fix-transform: rotate(180deg);
+  --sbb-options-panel-animation-transform: translateY(
+    calc(var(--sbb-options-panel-origin-height) / 2)
+  );
+}
+
+:host([disable-animation]:not([disable-animation='false'])) {
+  --sbb-options-panel-animation-duration: 0s;
+}
+
+::slotted(sbb-divider) {
+  margin-block: var(--sbb-spacing-fixed-3x);
+}
+
+:host([preserve-icon-space]:not([preserve-icon-space='false'])) {
+  --sbb-option-icon-container-display: block;
+}
+
+.sbb-select__trigger {
+  @include sbb.ellipsis;
+
+  height: calc(1em * var(--sbb-typo-line-height-body-text));
+}
+
+.sbb-select__trigger--placeholder {
+  color: var(--sbb-select-placeholder-color, var(--sbb-color-metal-default));
+  -webkit-text-fill-color: var(--sbb-select-placeholder-color, var(--sbb-color-metal-default));
```

Comment:
Yes, because the placeholder is not a real one (pseudo-element), but a span in the shadow-root, so we need to override the default color set by the sbb-form-field.

Reply:
Ok I see, can you comment it? It's a special case :)

Reply:
Done in 8aa86fbad0df7c12ec4985d704733ec7af1771aa